web/packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap

```diff
@@ -6903,7 +6903,7 @@ exports[`list of all events 1`] = `
           <td
             style="word-break: break-word;"
           >
-            User [joe] was denied access to Windows desktop [Administrator@100.104.52.89:3389] on [desktopaccess.com]
+            User [joe] was denied access to Windows desktop [Administrator@undefined] on [desktopaccess.com]
```

Comment:
Can we add `desktop_name` to these events in the story? `undefined` doesn't look too good 🙈

Reply:
Fixed :)